test/unit/profiles/profile_test.rb

```diff
@@ -108,7 +108,6 @@
       it 'prints loads of warnings' do
         logger.expect :info, nil, ["Checking profile in #{home}/mock/profiles/#{profile_id}"]
         logger.expect :error, nil, ["Missing profile version in inspec.yml"]
-        logger.expect :warn, nil, ["Missing profile title in inspec.yml"]
```

Comment:
I would expect that we see the result, if the inspec.yml is empty. Am I missing something?

Reply:
If the inspec.yml is missing, we generate a profile name.  Previously that name contained the path to the profile, which likely contained slashes.  As that's not permitted, we replaced the slashes; but to retain clarity, we now store the original path in the title.

So, a profile without metadata will now have both a name and a title generated, and thus there will be no warning about a missing title.

Discussion here: https://github.com/chef/inspec/pull/2231#discussion_r143976150

Reply:
@chris-rock this makes sense to me. It's only in the case where a profile exists, an inspec.yml exists, and the file is empty (or doesn't include a name). By default, we'll create a default name that includes the path (slugged), and a title that contains the actual path.

Since in the case of empty metadata / no name, we'll always have a title, then it's right to remove this test assertion.